crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -828,6 +828,30 @@ impl SuiTransactionBlockEvents {
     }
 }
 
+impl Display for SuiTransactionBlockEvents {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        if self.data.is_empty() {
+            writeln!(f, "╭───────────────────────────────────────────────────────────────────────────────────────────────────╮")?;
```

Comment:
Only question I have here is on the width of this table -- is that something that's always fixed by  `tabled` (in which case it's okay to hardcode that here), or is this going to look strange on narrow displays where `tabled` would have done something to account for that? In which case, you could use `tabled` to generate even this simple table.

Reply:
That's a great observation. The width of the table is dynamically driven by the inner data, as far as I understand, but in this case, when there are no `TransactionBlockEvents`, the little header there without any rows keeps the same length as the previous table (TransactionBlockEffects), which makes it look nicer. In general, I want to have a look on how we could do some automatic table sizing given the width of the terminal; tabled supports also wrapping rows by width, which I used in the past, but I need to collect sufficiently many examples of CLI usage (and odd/edge cases) to be able to have more meaningful requirements for this stuff. 

Nevertheless, you are right that we should probably not "force it" to be this long and instead have an empty table (which by default crashes tabled :)) ) I will update it!